docs/alps/vclusters.md

```diff
@@ -29,54 +22,74 @@ Host balfrin* daint* santis* todi*
 
 This would allow standard connections like `ssh santis` but also specifying the login node like `ssh santis-ln002` if needed. Replace `cscsusername` with your actual user name.
 
-## Daint
-
-Daint (Alps) is the vCluster dedicated to the User Lab. It is currently accessible at `daint.alps.cscs.ch` (until the current Piz Daint gets decommissioned), so connect with `ssh daint.alps` with the `ssh` settings above.
+## Santis
 
-Even though Weather and Climate also has the dedicated vCluster Santis (see [below](#santis)), traditional projects might also land on Daint.
+The vCluster `santis` is dedicated to **Climate and Weather**. It might, at the beginning, only host [EXCLAIM :material-open-in-new:](https://c2sm.ethz.ch/research/exclaim.html){:target="_blank"} and related projects.
 
-### Uenvs
+### Deployment Status
 
-List of currently supported Uenvs on Daint:
+By the end of 2024, the deployment is complete only to ~95 %. 
```

Comment:
```suggestion
By the end of 2024, the deployment will be approximately 95% complete.
```

Reply:
Actually I wanted to refer to the current state. So we could also write: "At the moment, ..."

Reply:
That would make more sense. Maybe something like "As of the current state (end of 2024), the deployment..."?